third-party/rsutils/include/rsutils/easylogging/easyloggingpp.h

```diff
@@ -41,17 +40,63 @@
+                    .construct( logger__ )                                                                             \
+                << STR;                                                                                                \
+        }                                                                                                              \
+    }                                                                                                                  \
+    while( false )
+
+// Same, but first convert to string using usual mechanisms
+#define LIBRS_LOG_( LEVEL, ... )                                                                                       \
+    do                                                                                                                 \
+    {                                                                                                                  \
+        auto logger__ = el::Loggers::getLogger( rsutils::g_librealsense_elpp_id );                                     \
+        if( logger__ && logger__->enabled( el::Level::LEVEL ) )                                                        \
+        {                                                                                                              \
+            std::ostringstream os__;                                                                                   \
+            os__ << __VA_ARGS__;                                                                                       \
+            el::base::Writer( el::Level::LEVEL, __FILE__, __LINE__, ELPP_FUNC, el::base::DispatchAction::NormalLog )   \
+                    .construct( logger__ )                                                                             \
+                << os__.str();                                                                                         \
+        }                                                                                                              \
+    }                                                                                                                  \
+    while( false )
+
+#define LOG_DEBUG(...)    LIBRS_LOG_( Debug,    __VA_ARGS__ )
+#define LOG_INFO(...)     LIBRS_LOG_( Info,     __VA_ARGS__ )
+#define LOG_WARNING(...)  LIBRS_LOG_( Warning,  __VA_ARGS__ )
+#define LOG_ERROR(...)    LIBRS_LOG_( Error,    __VA_ARGS__ )
+#define LOG_FATAL(...)    LIBRS_LOG_( Fatal,    __VA_ARGS__ )
+
+#define LOG_DEBUG_STR(STR)    LIBRS_LOG_STR_( Debug,    STR )
+#define LOG_INFO_STR(STR)     LIBRS_LOG_STR_( Info,     STR )
+#define LOG_WARNING_STR(STR)  LIBRS_LOG_STR_( Warning,  STR )
+#define LOG_ERROR_STR(STR)    LIBRS_LOG_STR_( Error,    STR )
+#define LOG_FATAL_STR(STR)    LIBRS_LOG_STR_( Fatal,    STR )
 
 namespace rsutils {
 
 
+extern std::string const g_librealsense_elpp_id;
```

Comment:
WHat is this? comment?

Reply:
g_ ?

Not self explained..

Reply:
It's the string representation of `LIBREALSENSE_ELPP_ID`. Internal, not for general use, and simply a performance optimization.